.vsts-ci.yml

```diff
@@ -21,7 +21,7 @@ parameters:
 - name: runTestBuild
   displayName: Run A Test Build
   type: boolean
-  default: false
+  default: true
```

Comment:
why is this set to true? it causes the publish stage to not run

Reply:
Ahh, another condition to fix. It was intended to keep around rolling CI.